extra/zabbix/vpoller-module/Makefile

```diff
@@ -1,2 +1,2 @@
 vpoller: vpoller.c
-	cc -fPIC -shared -o vpoller.so vpoller.c -I../../../include -I/usr/local/include -L/usr/local/lib -lzmq
+        cc -fPIC -shared -o vpoller.so vpoller.c -I../../../include/common -I../../../include -I/usr/local/include -L/usr/local/lib -lzmq
```

Comment:
Recipes should start with a tab, not spaces. Please fix this one.

Reply:
Visual Studio code replaced tab with spaces, fixed now.

Reply:
Used the gh web ide, now is fixed:
![image](https://github.com/user-attachments/assets/e138ed89-04f8-420b-a3ca-1e2230ec0e2d)